styles.css

```diff
@@ -0,0 +1,277 @@
+/*  Sign Up Button */
+body {
+    background: #eceef1;
+}
+div {
+    display: block;
+}
+.sign-up {
+    background: #eceef1;
+    /* background-color: chocolate; */
+    float: none;
+    padding: 10px 30px;
+}
+.s-u-section {
+    float: right;
+    height: 40px;
+    width: 300px;
+}
+.s-u-left {
+    /* background-color: white; */
+    float: left;
+    height: 36px;
+    margin-right: 10px;
+    margin-top: 5px;
+    text-align: right;
+    width: 170px;
+}
+.accnt-msg {
+    font-family: 'Nunito', sans-serif;
+    font-size: 12px;
+    height: 18px;
+    line-height: 1.5px;
+    margin-top: 2px;
+    padding: 1px;
+    text-align: center;
+    vertical-align: auto;
+    width: 190px;
+}
+.s-u-right {
+    float: left;
+    height: 34px;
+    margin-left: 10px;
+    text-align: right;
+    width: 102px;
+}
+.btn {
+    background: #0077C5;
+    border: 1px solid #4c4c4c;
+    border-radius: 4px;
+    color: white;
+    cursor: pointer;
+    font-family: 'Nunito', sans-serif;
+    font-size: 0.8em;
+    font-weight: normal;
+    height: 34px;
+    padding: 0 20px;
+    width: 100%;
+}
+.lock-img {
+    background-image: url(https://plugin.intuitcdn.net/identity-authn-core-ui/1.0.0-dev.52/images/icon-whitelock@3x.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: inline-block;
+    height: 1em;
+    margin-right: 4px;
+    margin-top: 2px;
+    vertical-align: text-top;
+    width: .8em;
+}
+
+/*Header*/
+
+.header-style {
+    text-align: center;
+    padding: 8px 0; 
+}
+.logo-container {
+    display: block;
+    margin: 0 auto;
+    width: 100%;
+}
+.logo-intuit {
+    height: 50px;
+    width: 150px;
+}
+
+/*Main Form*/
+
+section {
+    display: block;
+}
+.form-icons {
+    height: 35px;
+    margin: auto;
+    overflow: hidden;
+    padding-inline-start: 14px;
```

Comment:
`SUGGESTION`

Consider changing to `padding-left`, check this out with [css validator](http://jigsaw.w3.org/css-validator/validator) for the reason why you should.